eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
```

Comment:
i prefer using disabled for the component api and covering the type with propTypes.

Reply:
I think this helps because names should be clear enough to understand what the propTypes is without looking at the definition